ClinGen/clincoded
# Connects to #1771. Connects to #1777. Updates to GCI summaries and classification publication feature.

Steps to test #1771 (and #1777):
Required: classifications that were created before the recent segregation scoring changes.
To confirm the ability to generate an SOP v5 matrix:
1. Click on a "View Approved Summary" or "View Provisional Summary" button.  The resulting page should display the SOP version in use and a matching matrix.

To confirm the new approval process and publication limits on classifications that are not based on the current SOP version:
* Approve a provisional classification and attempt to publish it (the approval process should end after submitting the approval and no publish button/link will appear).
* Unpublish a published classification (no publish button/link will appear).
